wallet/src/de/schildbach/wallet/ui/dashpay/ContactsFragment.kt

```diff
@@ -124,7 +125,7 @@ class ContactsFragment : BottomNavFragment(R.layout.fragment_contacts_root), Tex
             }
         }
 
-        search_for_user.setOnClickListener {
+        search_for_user_suggestions.setOnClickListener {
```

Comment:
turns out we still need search_for_user.setOnClickListener, but I added it back in #611